norse/torch/functional/test/test_izhikevich.py

```diff
@@ -0,0 +1,464 @@
+import torch
+import numpy as np
+import norse.torch.functional.izhikevich as izk
+from norse.torch.functional.izhikevich import (
+    izhikevich_step,
+)
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+
+mpl.rcParams["axes.spines.left"] = False
+mpl.rcParams["axes.spines.right"] = False
+mpl.rcParams["axes.spines.top"] = False
+mpl.rcParams["axes.spines.bottom"] = False
+mpl.rcParams["xtick.top"] = False
+mpl.rcParams["xtick.bottom"] = False
+mpl.rcParams["xtick.labeltop"] = False
+mpl.rcParams["xtick.labelbottom"] = False
+mpl.rcParams["ytick.left"] = False
+mpl.rcParams["ytick.right"] = False
+mpl.rcParams["ytick.labelleft"] = False
+mpl.rcParams["ytick.labelright"] = False
+
+
+def plot_test():
```

Comment:
This test doesn't seem to assert anything. Would it make sense to put it somewhere else so it doesn't get picked up by pytorch? We could start a plotting module or something similar since we'd like to start that effort soon anyway.

Reply:
It's also a reeeeeaaly long function ;)

Reply:
Yes this was actually not really meant as a test but just as a way to figure out, whether the neuron did roughly what I expected it to do.

Reply:
And the code is relevant. I'm just wondering if we can exclude it from pytest's search pattern. This could actually likely be achieved by dropping the `test_` prefix